### PR TITLE
#17 superdev

### PR DESCRIPTION
**Pull request by SuperDev1990**
superdev

### CODE REVIEW

File: new_email.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ func main() {
 	)
 
 	to := []string{"bolt.shiny220@gmail.com"}
-	msg := []byte("To: devtech002snake@skiff.com\r\n" + 
+	msg := []byte("To: devtech003snake@skiff.com\r\n" +
```
Comment (devtech77snake) [inline new_email.go:17]: This is good.

**Review by devtech77snake — COMMENTED** [review]
okay?

File: new_email.go (hunk 1 above)
Comment (SuperDev1990) [reply new_email.go:17]: Yeah, it is great.

Comment (devtech77snake) [reply new_email.go:17]: Okay, it is resolved

**Review by devtech77snake — APPROVED** [review]
Good

**Comment by devtech77snake** [comment]
Your work is good.

**Pull request merged**